src/Node/DeprecatedNode.php

```diff
@@ -26,30 +26,36 @@ class DeprecatedNode extends Node
 {
     public function __construct(AbstractExpression $expr, int $lineno, ?string $tag = null)
     {
-        parent::__construct(['expr' => $expr], [], $lineno, $tag);
+        parent::__construct(['expr' => $expr], ['package' => 'twig/twig', 'version' => ''], $lineno, $tag);
```

Comment:
defaulting to `twig/twig` looks weird to me, as templates using that node are likely to belong to a different package.

And there is no way to provide those attributes from a `{% deprecated %}` tag in a template.

Reply:
Fixed